### DISCUSSION TITLE
When using Github for work, I only want to see employer-related items and updates

### DISCUSSION THREAD

**Discussion by rodneyfolz-asana** [community/community#68337] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi Feed team, thank you for accepting feedback publicly! My feedback, in the form of a user story:

* **As a** user of Github for my job/employer,
* **I want** to use my feed to keep up with my coworkers' changes to my company's repositories
* **So that** I can make more informed decisions about changes I make
* **By** better understanding the current state of change in our codebases

Before the algorithmic feed, I was able to do this by simply starring and following my coworkers and our repositories exclusively. Today, I have some control over the _types_ of stories in my feed, but not their _origin_.

My worry when using the filter function to remove stories is that I'll filter out stories of that type from my employer's repositories, which I do want to see.

I do like receiving recommended suggestions in my _personal account_'s feed. I'd even be open to feed suggestions on my work account, for my _employer's repositories and users_ that I'm not currently following.

However, because my current algorithmic feed pulls in content from all across Github, it's mostly noise when I am at work. As a result, I consider it a distraction from doing my job.

Thank you again for considering this feedback. Best, Rodney

**Comment by dpogue** [comment]
If you (or your company) pay for GitHub, and this is negatively impacting your ability to do your job, it might be worth opening a support ticket on https://support.github.com detailing how this is impacting you and specifically how you used the feed to do your job. Do **not** open tickets just to complain, but rather explain in detail your use case for the feed and how the new feed is falling short and how that's impacting your ability to be productive on GitHub (much like you did here in your post).
